tests/Unit/Domain/CoordinateMaps/Test_Wedge2D.cpp

```diff
@@ -150,43 +150,34 @@ void test_wedge2d_fail() noexcept {
 
 SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge2D.Fail", "[Domain][Unit]") {
   test_wedge2d_fail();
-}
-
-SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge2D.Equidistant",
-                  "[Domain][Unit]") {
-  test_wedge2d_all_orientations(false);
-}
-
-SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge2D.Equiangular",
-                  "[Domain][Unit]") {
-  test_wedge2d_all_orientations(true);
-}
-
-SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge2D.Equality",
-                  "[Domain][Unit]") {
-  const auto wedge2d =
-      CoordinateMaps::Wedge2D(0.2, 4.0, 0.0, 1.0, OrientationMap<2>{}, true);
-  const auto wedge2d_inner_radius_changed =
-      CoordinateMaps::Wedge2D(0.3, 4.0, 0.0, 1.0, OrientationMap<2>{}, true);
-  const auto wedge2d_outer_radius_changed =
-      CoordinateMaps::Wedge2D(0.2, 4.2, 0.0, 1.0, OrientationMap<2>{}, true);
-  const auto wedge2d_inner_circularity_changed =
-      CoordinateMaps::Wedge2D(0.2, 4.0, 0.3, 1.0, OrientationMap<2>{}, true);
-  const auto wedge2d_outer_circularity_changed =
-      CoordinateMaps::Wedge2D(0.2, 4.0, 0.0, 0.9, OrientationMap<2>{}, true);
-  const auto wedge2d_orientation_map_changed = CoordinateMaps::Wedge2D(
-      0.2, 4.0, 0.0, 1.0,
-      OrientationMap<2>{std::array<Direction<2>, 2>{
-          {Direction<2>::upper_eta(), Direction<2>::upper_xi()}}},
-      true);
-  const auto wedge2d_use_equiangular_map_changed =
-      CoordinateMaps::Wedge2D(0.2, 4.0, 0.0, 1.0, OrientationMap<2>{}, false);
-  CHECK_FALSE(wedge2d == wedge2d_inner_radius_changed);
-  CHECK_FALSE(wedge2d == wedge2d_outer_radius_changed);
-  CHECK_FALSE(wedge2d == wedge2d_inner_circularity_changed);
-  CHECK_FALSE(wedge2d == wedge2d_outer_circularity_changed);
-  CHECK_FALSE(wedge2d == wedge2d_orientation_map_changed);
-  CHECK_FALSE(wedge2d == wedge2d_use_equiangular_map_changed);
+  test_wedge2d_all_orientations(false);  // Equidistant
+  test_wedge2d_all_orientations(true);   // Equiangular
+  SECTION("Equality") {
```

Comment:
SECTION reruns the test from the beginning. Is that really what you want here? I think you can just use scope + a comment. You could also put all the blocks into SECTIONs like you've done in other tests.